src/main/java/koreatech/in/domain/User/owner/Owner.java

```diff
@@ -41,4 +43,25 @@ public void enrichAuthComplete() {
         setUser_type(UserType.OWNER);
         setIs_authed(false);
     }
+
+    public boolean hasRegistrationInformation() {
+        return this.company_registration_number != null;
+    }
+
+    public static boolean hasRegistrationInformation(OwnerRegisterRequest request) {
+        return hasCompanyNumber(request) && hasAttachmentUrls(request);
+    }
+
+    public static boolean isInsufficientRegisterRequest(OwnerRegisterRequest request) {
+        return hasCompanyNumber(request) ^ hasAttachmentUrls(request);
+    }
```

Comment:
```suggestion
    public oolean isSufficientRegisterRequest(OwnerRegisterRequest request) {
        return !(hasCompanyNumber(request) ^ hasAttachmentUrls(request));
    }
```
처럼 긍정형으로 변경 + static 제거는 어떨까요?

Reply:
적용했습니다